urlmatcher.py

```diff
@@ -83,6 +83,10 @@ def url_matcher(self, msg, match):
         else:
             description = readable_article
 
-        return "~> {}\n~> {}\n~> {}".format(url,
+        if description:
+            return "~> {}\n~> {}\n~> {}".format(url,
                                             readable_title,
                                             description)
+        else:
+            return "~> {}\n~> {}".format(url,
+                                     readable_title) 
```

Comment:
@Krista Notice that we try to make sure that we are consistent with indents. In the diff on the left you can see that `url`, `readable_title` and `description` are on the same level inside the `format` function.

Since Python tries to be a very readable language, there exists a standard that described what Python should looks like. It is called [PEP8](https://www.python.org/dev/peps/pep-0008/).

Many issues like this can also be found automatically using various utilities. I personally recommend `flake8` which you can install by running

```bash
$ pip install flake8
```

Running `flake8` on the file you changed returns

```
urlmatcher.py:88:45: E128 continuation line under-indented for visual indent
urlmatcher.py:89:45: E128 continuation line under-indented for visual indent
urlmatcher.py:92:38: E128 continuation line under-indented for visual indent
urlmatcher.py:92:53: W291 trailing whitespace
```